pennylane_lightning/core/bindings/Bindings.cpp

```diff
@@ -1,91 +0,0 @@
-// Copyright 2018-2024 Xanadu Quantum Technologies Inc.
```

Comment:
Would it maybe be better to rename all of the `_nb` files to match these old filenames? For example, after removing the pybind11 `Bindings.hpp`, rename `Bindings_nb.hpp` to `Bindings.hpp`